### ISSUE TITLE
Add language info from census to other demographics

### ISSUE THREAD

**Issue by allenbaron** [DataKind-DC/EcoAction#2]
Language stats are missing from `tidycensus` data obtained for Arlington county census block groups and needs to be added.

**Comment by icanhazcodeplz** [comment]
As far as I can tell, ACS does not collect language data at either the block-group level or the tract level. We might be out of luck on this one.

**Comment by allenbaron** [comment]
I've only been able to find the information to the county level but a [study on Arlington County regarding language spoken](https://projects.arlingtonva.us/wp-content/uploads/sites/31/2014/03/ACS-Language.pdf) using earlier 5-yr ACS data suggests it should be.

The data on that pdf suggests only about 2% of Arlington County is linguistically isolated with only slight variation within the county. The total linguistic isolation and the variation within the county is small enough that I'd agree it doesn't need to be included in a model/map. We can point EcoAction to this pdf (if they aren't already aware of it), in case they want to reach out to some of these households.

Unless @timcova already has this info, let's not worry about it. I'll leave this open until we get a response from him about it.

**Comment by timcova** [comment]
I do have the data.  I had the same issue with trying to download it from the census.gov site.  The lowest-level data I could actually access was aggregated at the county level.  I ended up getting it from IPUMS ( Integrated Public Use Microdata Series, associated with the University of Minnesota, which hosts the ACS data, and a variety of other data sets.  https://ipums.org/)  I got the language distribution data, along with a variety of other related/semi-related socio-economic data (such as Housing Units, Occupancy Status of Housing Units,  Tenure in Occupied Housing Units,  Households by Presence of Nonrelatives, etc) at the block-group level for 2010 and 2018.  For good, or bad, I ended up downloading the block group data for the entire state of Virginia.  Tasks to do are separating out just the language data (Source code: C16002, NHGIS code:  AJY2) from the other data columns, and separating out the Arlington block group data from the entire set of Virginia block group data.  I will try to do that this weekend.  I have also attached nhgis0001_csv.zip, if anyone wants to start working with the data immediately.
[nhgis0001_csv.zip](https://github.com/DataKind-DC/EcoAction/files/5969442/nhgis0001_csv.zip) The data is in the .csv files, organized by data value type (language, occupied, etc) in columns across, and county-tract-block group down in rows.  The .txt files provide the lookup between the column headings and the text descriptions of the data values.  nhgis0001_ds239_20185_2018_blck_grp is the most complete set of data, and includes the language data (AJY20001-AJY20014 for the percentages by language, languages below),  and all of the other columns.

        AJY2E001:    Total
        AJY2E002:    English only
        AJY2E003:    Spanish
        AJY2E004:    Spanish: Limited English speaking household
        AJY2E005:    Spanish: Not a limited English speaking household
        AJY2E006:    Other Indo-European languages
        AJY2E007:    Other Indo-European languages: Limited English speaking household
        AJY2E008:    Other Indo-European languages: Not a limited English speaking household
        AJY2E009:    Asian and Pacific Island languages
        AJY2E010:    Asian and Pacific Island languages: Limited English speaking household
        AJY2E011:    Asian and Pacific Island languages: Not a limited English speaking household
        AJY2E012:    Other languages
        AJY2E013:    Other languages: Limited English speaking household
        AJY2E014:    Other languages: Not a limited English speaking household
